client/src/components/Calendar/CalendarViewParent.tsx

```diff
@@ -0,0 +1,134 @@
+import { Button } from 'flowbite-react';
+import { useState } from 'react';
+import { CalendarViewEnum } from '../../enums';
+import { capitalizeWord } from '../../utils/functions';
+import { FaAngleDown, FaAngleLeft, FaAngleRight } from 'react-icons/fa6';
+
+interface Props {}
+
+const CalendarViewParent = () => {
+  const [selectedView, setSelectedView] = useState<string>(
+    localStorage.getItem('calendarView') || CalendarViewEnum.WEEK
+  );
+  const [isDropdownOpen, setIsDropdownOpen] = useState<boolean>(false);
+
+  const handleViewSelection = (calendarView: CalendarViewEnum) => {
+    localStorage.setItem('calendarView', calendarView);
+    setSelectedView(calendarView);
+    setIsDropdownOpen(false);
+  };
+
+  const horizontalDotsSVG = (
+    <svg
+      className="h-5 w-5"
+      aria-hidden="true"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="currentColor"
+      viewBox="0 0 16 3"
+    >
+      <path d="M2 0a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Zm6.041 0a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3ZM14 0a1.5 1.5 0 1 1 0 3 1.5 1.5 0 0 1 0-3Z" />
+    </svg>
+  );
```

Comment:
Is it possible we can maybe just import the react-icons version instead? If it exists?

Reply:
I tried but didn't find one that looks as good imo. The react icon version of it is way smaller and barely visible.